components/ShortFactsBox/index.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+
+const FactBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 5px;
+  // margin: 10px;
+  border-radius: 20px;
+`;
+const ShortFact = styled.span`
+  background-color: #858f87;
+  border-radius: 20px;
+  padding: 5px;
+  color: white;
+  padding-left: 10px;
+  padding-right: 10px;
+`;
+
+export default function ShortFactsBox({ project }) {
+  return (
+    <FactBox>
+      <ShortFact>{project.time} hours</ShortFact>
+      <ShortFact>{project.priceCategory}</ShortFact>
+      <ShortFact>{project.category}</ShortFact>
+      <ShortFact>{project.difficulty}</ShortFact>
```

Comment:
This looks like a job for `.map` 🦸 !

Reply:
... [but it _could be_ an array with `Object.keys()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys), which you could map over. Usually when something repeats itself fairly consistently, it can be handled a little bit more beautifully. It's not a must, if you don't feel like doing, but I'd consider it decent practice as far as code quality is concerned.

Reply:
Thank you for the explanation. We tried it out and came up with this:

      {Object.keys(project).map((key) => {
        if (["time", "priceCategory", "category", "difficulty"].includes(key))
          return <ShortFact key={project.id}>{project[key]}</ShortFact>;
      })}

Since we don't want **all** of the values in the object to be displayed as a short fact we would have to filter additionally. And also, e.g. the time-information needs to be suffixed with the string "hours". This is also not possible with the map-method. So it seems to us that the original solution is more efficient in this case. Thank you anyways :)